docs/UsersGuide/source/RunSRW.rst

```diff
@@ -0,0 +1,1239 @@
+   workflow_switches:
+      RUN_TASK_GET_OBS_CCPA: false
+      RUN_TASK_GET_OBS_MRMS: false
+      RUN_TASK_GET_OBS_NDAS: false
+
+If users have access to NOAA :term:`HPSS` but have not pre-staged the data, they can simply set the ``RUN_TASK_GET_OBS_*`` tasks to true, and the machine will attempt to download the appropriate data from NOAA HPSS. In this case, the ``*_OBS_DIR`` paths must be set to the location where users want the downloaded data to reside. 
+
+Users who do not have access to NOAA HPSS and do not have the data on their system will need to download :term:`CCPA`, :term:`MRMS`, and :term:`NDAS` data manually from collections of publicly available data, such as the ones listed `here <https://dtcenter.org/nwp-containers-online-tutorial/publicly-available-data-sets>`__. 
+
+Next, the verification tasks must be turned on according to the user's needs. Users should add some or all of the following tasks to ``config.yaml``, depending on the verification procedure(s) they have in mind:
+
+.. code-block:: console
+
+   workflow_switches:
+      RUN_TASK_VX_GRIDSTAT: true
+      RUN_TASK_VX_POINTSTAT: true
+      RUN_TASK_VX_ENSGRID: true
+      RUN_TASK_VX_ENSPOINT: true
+
+These tasks are independent, so users may set some values to true and others to false depending on the needs of their experiment. Note that the ENSGRID and ENSPOINT tasks apply only to ensemble model verification. Additional verification tasks appear in :numref:`Table %s <VXWorkflowTasksTable>`. More details on all of the parameters in this section are available in :numref:`Section %s <VXTasks>`. 
+
+.. _GenerateWorkflow: 
+
+Generate the Regional Workflow
+-------------------------------------------
+
+Run the following command from the ``ufs-srweather-app/ush`` directory to generate the workflow:
+
+.. code-block:: console
+
+   ./generate_FV3LAM_wflow.py
+
+The last line of output from this script, starting with ``*/1 * * * *`` or ``*/3 * * * *``, can be saved and :ref:`used later <Automate>` to automatically run portions of the workflow if users have the Rocoto workflow manager installed on their system. 
+
+This workflow generation script creates an experiment directory and populates it with all the data needed to run through the workflow. The flowchart in :numref:`Figure %s <WorkflowGeneration>` describes the experiment generation process. First, ``generate_FV3LAM_wflow.py`` runs the ``setup.py`` script to set the configuration parameters. Second, it copies the time-independent (fix) files and other necessary data input files from their location to the experiment directory (``$EXPTDIR``). Third, it copies the Weather Model executable (``ufs_model``) from the ``exec`` directory to ``$EXPTDIR`` and creates the input namelist file ``input.nml`` based on the ``input.nml.FV3`` file in the ``parm`` directory. Lastly, it creates the workflow XML file ``FV3LAM_wflow.xml`` that is executed when running the experiment with the Rocoto workflow manager.
```

Comment:
@gspetro-NOAA You may want to re-word it a bit because:
- Fix files are sym-linked by default now instead of copied
- The ``ufs_model`` binary is not copied to ``EXPTDIR`` as far as I know

Reply:
@gspetro-NOAA As @danielabdi-noaa said, the fix files are no longer being copied, but are being sym-linked.  Here is an example of the fix_am on Cheyenne:

`fix_am -> /glade/p/ral/jntp/UFS_SRW_App/develop/fix/fix_am`

Looking through the contents of `$EXPTDIR`, I don't see the `ufs_model` binary.  So, I don't think that it is copied or sym-linked to `$EXPTDIR`.  In the `var_defns.sh` file in the `$EXPTDIR`, I do see the following:

`FV3_EXEC_FP='/glade/scratch/mlueken/ufs-srweather-app/exec/ufs_model'`

This would suggest to me that the ufs_model executable is not copied to the `$EXPTDIR`, but launched directly from the `exec` directory.

Reply:
Okay, I fixed to reflect that! :)